modules/jwplayerRtdProvider.js

```diff
@@ -235,6 +237,9 @@ export function getVatFromPlayer(playerID, mediaID) {
   };
 }
 
+/*
+  deprecated
+ */
```

Comment:
What's the reasoning to mark this deprecated when it's still being called above? Not a big deal, just curious.

Reply:
@Rothalack it's being called because we don't want to break the API. The call creates a targeting object which is set to`bid.rtd.jwplayer.targeting` which we are deprecating in favor of `ortb2.site`. We will remove the call once no SSP reads from `bid.rtd.jwplayer`